models/DataObject/Concrete.php

```diff
@@ -594,9 +594,16 @@ public function getValueFromParent($key, $params = null)
      */
     public function getNextParentForInheritance()
     {
+        return $this->getClosestParentOfClass($this->getClassId());
+    }
+
+    /**
```

Comment:
@param string $classId

Reply:
Added it. 
In my own projects I do not add param / return doctypes when this is already present via typehints. Is there a general rule for Pimcore to always add param / return doctypes - also if they do not add any value?